depends/packages/qt.mk

```diff
@@ -47,16 +47,15 @@ $(package)_config_opts += -no-ico
 $(package)_config_opts += -no-iconv
 $(package)_config_opts += -no-kms
 $(package)_config_opts += -no-linuxfb
-#$(package)_config_opts += -no-libjpeg
+$(package)_config_opts += -no-libjpeg
```

Comment:
nit: why is 6e5440d505d351e95266e322ee73aff7fa0e72b4 not squashed into previous?